web/new_alt_text_manager.js

```diff
@@ -274,6 +288,45 @@ class NewAltTextManager {
     this.#textarea.value = altText;
   }
 
+  #setProgress() {
+    // Show the download model progress.
+    this.#downloadModel.classList.toggle("hidden", false);
+
+    const callback = async ({ detail: { finished, total, totalLoaded } }) => {
+      const one_mega_bytes = 1e6;
```

Comment:
Since this is a constant, this should be upper-case.
```suggestion
      const ONE_MEGA_BYTES = 1e6;
```